substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1990,6 +1990,10 @@ impl<T: Config> StakingInterface for Pallet<T> {
 		T::BondingDuration::get()
 	}
 
+	fn max_bonding_duration(_value: Self::Balance) -> EraIndex {
```

Comment:
should be removed.

Reply:
Removed in [0f7d125](https://github.com/paritytech/polkadot-sdk/pull/8298/commits/0f7d12504cd6620004e2107ba8d08ab67e3f3861).